Eda 07 sep 2023

MatiMatlakala opened the pull request:
Please review the additions to the notebook and add changes where required.

MatiMatlakala committed:
Merge pull request #1 from TeamLavender001/main

First notebook draft

MatiMatlakala committed:
EDA Process

The following were added to the notebook:
Data inspection.
Data cleaning, processing and analysis.

MatiMatlakala committed:
Merge branch 'EDA-07-Sep-2023' of https://github.com/TeamLavender001/Lavender into EDA-07-Sep-2023

MatiMatlakala committed:
EDA Process

The following processes were made:
Data inspection.
Data cleaning, processing and analysis.

majoralaro reviewed (commented):
For the second point on the insights pertaining to the Sales per city, can we compare San Fransico to other cities instead of comparing it to Austin only?  This would suggest that San Fransico has the highest cost of living. 

It seems that the first two insights on the correlation coefficient are repetitive.

MatiMatlakala committed:
Final edits

- Removed a sentence from the correlation chart as it was repetitive.
- Added a comma in the references.
